src/databricks/labs/lsql/backends.py

```diff
@@ -178,6 +179,8 @@ def _row_to_sql(row: DataclassInstance, fields: tuple[dataclasses.Field[Any], ..
                 data.append(f"'{value}'")
             elif field_type == int:
                 data.append(f"{value}")
+            elif field_type == dict[str, str]:
```

Comment:
Is it really that simple? Please add more test cases, where you have dict[int,str], dict[int,list[str]], dict[str,dict[str,str]]